components/eam/bld/config_files/definition.xml

```diff
@@ -323,9 +326,6 @@ Switch to enable or disable MMF variance transport: 0=off, 1=on.
 <entry id="use_ECPP" valid_values="0,1" value="0">
 Switch to enable or disable ECPP in CAM: 0=off, 1=on.
 </entry>
-<entry id="use_MAML" valid_values="0,1" value="0">
-Switch to enable or disable MAML in CAM: 0=off, 1=on.
-</entry>
 <entry id="use_crm_cldfrac" valid_values="0,1" value="0">
 Switch to enable or disable fractional cloudiness in m2005 microphysics in CRM: 0=off, 1=on.
 </entry>
```

Comment:
Just so you are moidifying definition.xml. Do you want to change SPCAM to MMF in the description of 'MMF_microphysics_scheme", "crm","crm_adv"?

Reply:
good catch, fixed.